modules/installation-guide/partials/proc_enabling-dev-workspace-operator.adoc

```diff
@@ -32,18 +32,25 @@ spec:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:deploy --che-operator-cr-patch-yaml=patch.yaml ...
+$ {prod-cli} server:deploy --workspace-engine=dev-workspace ...
```

Comment:
if it's merged into 7.42 this flag won't be needed as devWorkspace will be the default

Reply:
does it break anything? tbh I'm not sure about the release process so I don't know where exactly this will land if I merge today.